src/framework/accordionItem/AccordionItem.jsx

```diff
@@ -10,12 +10,32 @@ export default class AccordionItem extends Component {
   constructor(props) {
     super(props);
 
-    this.onTitleClick = this.props.onTitleClick.bind(this);
+    this.state = {
+      isActive: props.isActive,
+    };
+
+    this.onTitleClick = this.onTitleClick.bind(this);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({
```

Comment:
Could do only when `this.state.isActive !== nextProps.isActive` to prevent too many re-renderings.

Reply:
Good idea!